docs/docs/reference/builtin-functions/feel-built-in-functions-boolean.md

```diff
@@ -55,3 +55,26 @@ is defined(x)
 is defined(x.y)
 // false - if no variable "x" exists or it doesn't have a property "y"
 ```
+
+## get or else(value, default)
+
```

Comment:
:x: We should mark this function as Camunda extension (i.e. not part of the DMN spec).

We can add the marker by the following snipped, similar to the function above. :wink: 

```suggestion

<MarkerCamundaExtension></MarkerCamundaExtension>

```